motoman_mh110_support/package.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_mh110_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman MH110 (and variants).</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman MH110 series of manipulators.
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>MH110 - Default</li>
+    </ul>
+    <p>
+      Visual appearance, joint limits and maximum joint velocities are 
+      identical to the successor GP110 model, the only differences being
+      joint effort limits and DX200 vs YRC1000 controller connection.
+      Information on the MH110 can be found in <em>HW1482741 (169859-1CD Rev. 2)</em>
+      at the <a href="https://www.motoman.com/en-us/service-training/product-documentation">Yaskawa Motoman product documentation site</a>.
```

Comment:
```suggestion
      Information on the MH110 can be found in <em>HW1482741 (169859-1CD Rev. 2)</em>.
```

Reply:
It's clear this is a reference to a Yaskawa Motoman released document. That should be sufficient to find it again if needed.